features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -121,6 +123,7 @@ Feature: Gift Exchange Challenge
     Given the gift exchange "Awesome Gift Exchange" is ready for signups
       And I am logged in as "myname1"
     When I sign up for "Awesome Gift Exchange" with combination A
+      And all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.